chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/summarystatistics/EditionCriteria.java

```diff
@@ -0,0 +1,5 @@
+package gov.healthit.chpl.scheduler.job.summarystatistics;
+
+public enum EditionCriteria {
+    ALL, EDITION_2011, EDITION_2014, EDITION_2015_CURES, EDITION_2015_NON_CURES, EDITION_2015_CURES_AND_NON_CURES
```

Comment:
Worth putting these on multiple lines so when we (inevitably) need to change them it's easier to read the line diff?